news/2022-08-30-changes-to-osu-sr-and-pp.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: "Changes to osu! Star Rating & Performance Points"
+date: 2022-08-30 12:00:00 +0000
+
+---
+
+Exciting times ahead as a new wave of difficulty calculation changes reach the shores of the osu! game mode!
+
+The time for changes has arrived yet again. Developers have been hard at work since the [last wave of changes](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates) to improve the accuracy of difficulty calculations, and to address community concerns. This newspost will explore the list of changes, explaining things along the way and informing you - the lovely player!
+
+As always, a wealth of information can be accessed via the [star rating](https://osu.ppy.sh/wiki/en/Beatmap/Star_rating) and [performace points](https://osu.ppy.sh/wiki/en/Performance_points) wiki pages for those that want background reading.
+
+---
+
+## osu! changes
+
+As a reminder of the various core values in the difficulty calculation algorithm:
+
+- **Aim**: how difficult it is to consistently hit notes back-to-back (affected by approach rate, jumps, Hidden, Hard Rock and more)
+- **Speed**: how quickly new elements occur to the player (affected by note density, BPM, Double Time and Half Time)
+- **Accuracy**: how difficult it is to stay rhythmically on time (affected by Hidden, Hard Rock, Flashlight and more)
+- **Flashlight**: how difficult it is to memorise and hit every object in a map with the Flashlight mod enabled
+- **Strain**: the difficulty of enduring intense or technically demanding patterns often and for a long time (affected by streams, fast jumps and more)
+
+### Worst-case scenario accuracy scaling for Speed
+
+Currently, performance calculation treats every object in a map equally when weighing accuracy. Maps which feature a lot of filler notes between tough sections unfairly benefit from this, as accuracy will be scaled across the whole map rather than only on the difficult sections. This is now going to change, following [a change](https://github.com/ppy/osu/pull/15035) administered by [emu1337](https://osu.ppy.sh/users/2185987)!
+
+The number of notes that are considered difficult by Speed calculations will now be counted. Accuracy will then be scaled by the average between this count and the overall object count. This means that for those maps with filler, the punishment from 100s/50s/misses will now be more accurate. The averaging is done for the sake of leniency - using the Speed count alone resulted in values that were too harsh in testing.
```

Comment:
I think the last sentence might be necessary: I added it because some of the comments I saw from people randomly (stuff like "it should be somewhat lenient at least"). Adding Hidamari as an example sounds like a good idea.

Reply:
Was thinking along the lines of "the punishment from inaccuracies will now be more accurate to the composition of the map", so I might need to make the wording more explicit or rephrase entirely? Happy to take suggestions here